services/webapp/src/features/posts/containers/AuthorDetails.jsx

```diff
@@ -32,11 +32,11 @@ const dispatch2props = {
 
 const Loading = () => null
 
-const Content = ({ name, email, address }) => (
+const Content = ({ id, name, email, address }) => (
     <div>
-        <PageTitle>{name}</PageTitle>
+        <PageTitle><Link to={`/users/${id}/details`}>{name}</Link></PageTitle>
         <div>
-            <a href={`mailto:${email}`}>{email}</a>
+            <Link to={`mailto:${email}`}>{email}</Link>
```

Comment:
this link does not work, try it out yourself.

I guess this has to be a normal anchor tag as is the browser that takes care of the `mailto:` part, so to launch the mail client.